packages/host/app/components/ai-assistant/toast.gts

```diff
@@ -134,7 +133,7 @@ export default class AiAssistantToast extends Component<Signature> {
     const state: {
       value: {
         roomId: string;
-        message: MessageField;
+        message: RoomMessageModel;
```

Comment:
Can this name be simplified to `RoomMessage`? And other names with the model suffix... perhaps this is unnecessary?